src/storage/mongodb/SettingStorage.ts

```diff
@@ -503,4 +503,16 @@ export default class SettingStorage {
     }
     return SettingStorage.saveSettings(tenant, setting);
   }
+
+  public static async saveAssetSettings(tenant: Tenant, assetSettings: AssetSettings): Promise<string> {
```

Comment:
Should look like saveUserSettings() and saveCryptoSettings()...
Don't reinvent the wheel and follow the same principles that we have used so far even if they can be improved (this would be another issue in that case.)